### DISCUSSION TITLE
Pinecil not staying connected

### DISCUSSION THREAD

**Discussion by mrw1986** [builder555/PineSAM#215] [Q&A]
I'm using the binary version of PineSAM on Windows 11 and I'm unable to keep my Pinecil connected. I ensured it's allowed in the firewall, unblocked the zip file, etc. I've tried restarting to no avail. I've attached an image of the PowerShell output. Any help is appreciated!

![image](https://github.com/user-attachments/assets/367cd0cd-fcae-4e4e-a0b0-fd0039d40c8f)

**Comment by builder555** [comment]
Just to clarify, you can connect at first, but it disconnects? or it never connects in the first place?

**Comment by mrw1986** [comment]
It connects for a second or so and then disconnects.

**Comment by builder555** [comment]
I'm still trying to get a hold of a win11 machine, it's tough debugging just by looking at code.

**Comment by mrw1986** [comment]
No problem, I appreciate you even looking at it. Thanks!

**Comment by builder555** [comment]
which firmware version does your pinecil have?

**Comment by mrw1986** [comment]
2.22

**Comment by builder555** [comment]
I borrowed a friend's laptop. I am able to see the disconnect error, but after I refresh the webpage, it connects and seems to be working, is that also the case for you? 
I'm still looking into why it disconnects in the first place.

**Comment by mrw1986** [comment]
I'll have to double check when I have a chance. My family and I are dealing
with something currently and I've been too busy. I can let you know
hopefully within the next few days. Thanks!

**Comment by builder555** [comment]
no problem, take your time!
